### PR TITLE
Update-translations | #393 

### LINKED ISSUE #393: ! alle Informationen über den diesjährigen BreakOut aktualisieren
alle Informationen über den diesjährigen BreakOut aktualisieren, dies muss im Code angepasst werden: 
	Spendenpartner: Zeltschule e.V. 
	Anmeldezeitraum: 14.04.- 31.5.2022
	BreakOut-Datum: 24.6./25.6.2022
 Ende des Überweisungzeitraums: 31.5.2022
	(Hilfestellung: wir haben letztes Jahr mit Strg+f im Code geschaut, wo überall der letztjährige Spendenpartner, sowie die oben genannten Daten genannt werden. Letztjähriger Spendenpartner: “Viva con Agua” oder “Viva con Agua de Sankt Pauli e.V.” oder “VcA”. Datum letzter BreakOut: 19./20.6.2021, Datum letzte Anmeldephase: 26.4.-24.5.2021)

### PR DESCRIPTION
**Pull request by abatmas**

### CODE REVIEW

File: src/common/resources/translations/translations.en.js [hunk 1]
```diff
@@ -388,7 +388,7 @@ module.exports = {
     "OPTION_SHIRT_2": "Size M",
     "OPTION_SHIRT_3": "Size L",
     "OPTION_SHIRT_4": "Size XL",
-    "PAYMENT_INFO": "The payment of your team participation fee (50,-€) has not yet reached us. Please transfer the amount by 24th of May 2021 to participate in the event.",
+    "PAYMENT_INFO": "The payment of your team participation fee (50,-€) has not yet reached us. Please transfer the amount by 31th of May 2022 to participate in the event.",
```
Comment (SimonK92) [inline src/common/resources/translations/translations.en.js]: 31th -> 31st

File: src/common/resources/translations/translations.en.js [hunk 2]
```diff
@@ -407,8 +407,8 @@ module.exports = {
...
-    "DESCRIPTION_4": "So the total amount per term is 50€ (15€ Registration per person + 20€ Deposit). For a succesful registration the total amount has to be transferred by 24th May 2021. Please use the <strong>mentioned purpose</strong> and the following bank account:",
+    "DESCRIPTION_3": "Since the idea behind BreakOut is to give donations to Zeltschule e.V., we want to make sure that every team supports this social idea. Therefore you are required to give a deposit of 20€. You will get it back after the event, if your team has generated more than 100€ in donations. Otherwise the 20€ will go directly to Zeltschule e.V. in your name. But don't worry, the goal is easily reached! <3",
+    "DESCRIPTION_4": "So the total amount per term is 50€ (15€ Registration per person + 20€ Deposit). For a succesful registration the total amount has to be transferred by 31th May 2022. Please use the <strong>mentioned purpose</strong> and the following bank account:",
```
Comment (SimonK92) [inline src/common/resources/translations/translations.en.js]: 31th -> 31st

File: src/common/resources/translations/translations.en.js [hunk 3]
```diff
@@ -548,7 +548,7 @@ module.exports = {
     "SERVER_PROCESSING_REQUEST":"Medium is being processed by the server, give us a moment!",
     "POINTS": "Points",
     "TEAM_FEE_HEADING": "Thank you for signing up for BreakOut!",
-    "TEAM_FEE_DESCRIPTION": "To complete your registration, please transfer your registration fee as soon as possible. We can only activate your team once we've received the registration fee. Please transfer the fee by latest May 24th 2021 so that we can equip you with a t-shirt and your starter kit.",
+    "TEAM_FEE_DESCRIPTION": "To complete your registration, please transfer your registration fee as soon as possible. We can only activate your team once we've received the registration fee. Please transfer the fee by latest May 31th 2022 so that we can equip you with a t-shirt and your starter kit.",
```
Comment (SimonK92) [inline src/common/resources/translations/translations.en.js]: 31th -> 31st

**Review by SimonK92 — COMMENTED** [review]
31th -> 31st

**Review by sibbl — APPROVED** [review]

**Pull request merged**